x-pack/plugins/apm/server/utils/offset_previous_period_coordinate.ts

```diff
@@ -8,6 +8,26 @@
 import moment from 'moment';
 import { Coordinate } from '../../typings/timeseries';
 
+function getDateDiff({ start, end }: { start: number; end: number }) {
+  return moment(start).diff(moment(end));
+}
+
+export function offsetXCoordinate({
```

Comment:
why do we need all this? what's the difference between this and `const offset = (start - previousStart)`?